### PR TITLE
Subsystem/grabber

### PR DESCRIPTION
**Pull request by Hannah3333**
Wrote code for the grabber--absorbing the cube, sensing once the cube has been absorbed, and spitting out the cube.

### CODE REVIEW

File: subsystems/grabber.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import wpilib
+
+class Grabber:
+	"""
+		The grabber will absorb a cube,
```
Comment (porglezomp) [inline subsystems/grabber.py]: The docstring shouldn't be indented like this

**Review by porglezomp — CHANGES_REQUESTED** [review]
Needs different whitespace

**Review by helenaheinonen — APPROVED** [review]
looks like my cat which means its good✨ 🐱

**Review by PCain21 — APPROVED** [review]
looks good

**Review by porglezomp — APPROVED** [review]
LGTM

**Review by porglezomp — CHANGES_REQUESTED** [review]
Actually, let's get some tests on this. Merge from `develop` into your branch, and write a test in the `tests` folder to make sure these do what we expect.

File: tests/grabber_test.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    grabber.spit()
+    assert left_motor.state == -1
+    assert right_motor.state == 1
+    grabber.spit(speed=0.5)
+    assert left_motor.state == -0.5
+    assert right_motor.state == 0.5
+    assert not grabber.has_cube()
+    sensor.state = True
+    assert grabber.has_cube()
+
```
Comment (porglezomp) [inline tests/grabber_test.py:34]: In the future keep the end clean, not so many blank lines

File: tests/grabber_test.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    grabber.absorb(speed=0.5)
+    assert left_motor.state == 0.5
+    assert right_motor.state == -0.5
+    grabber.spit()
+    assert left_motor.state == -1
+    assert right_motor.state == 1
+    grabber.spit(speed=0.5)
+    assert left_motor.state == -0.5
+    assert right_motor.state == 0.5
+    assert not grabber.has_cube()
```
Comment (porglezomp) [inline tests/grabber_test.py:31]: In the future, it's nice to separate the different sections with extra space, like
```python
grabber.absorb(speed=0.5)
assert left_motor.state == 0.5
assert right_motor.state == -0.5

grabber.spit()
assert left_motor.state == -1
assert right_motor.state == 1
```

**Review by porglezomp — APPROVED** [review]
You could format it nicer in the future, but this is good.

**Pull request merged**